NEWS.md

```diff
@@ -7,6 +7,7 @@ This release includes FQM Integration, separate notes by note type, updating Sup
 * [UXPROD-3903](https://folio-org.atlassian.net/browse/UXPROD-3903) - Support `data-export-spring` interface 2.0. Breaking change in `data-export-spring` does not impact mod-bulk-operations use
 * [MODBULKOPS-197](https://issues.folio.org/browse/MODBULKOPS-197) - Accommodate changes in holdings records schema
 * [MODBULKOPS-181](https://issues.folio.org/browse/MODBULKOPS-181) - Always include column with edited properties on Are you sure form
+* [MODBULKOPS-361](https://folio-org.atlassian.net/browse/MODBULKOPS-361) Update `data-import-converter-storage` to `v1.5`, `source-manager-job-executions` to `v3.4`, `metadata-provider-job-execution-logs` to `v1.1`
```

Comment:
Looks like this entry should not be here as this block belongs to already released version. Please remove it, I believe the news file will be updated at the time of release